pkg/adapter/v2/cloudevents.go

```diff
@@ -23,19 +24,39 @@ import (
 	"net/url"
 	"time"
 
-	cloudeventsobsclient "github.com/cloudevents/sdk-go/observability/opencensus/v2/client"
+	obshttp "github.com/cloudevents/sdk-go/observability/opencensus/v2/http"
 	cloudevents "github.com/cloudevents/sdk-go/v2"
+	ceclient "github.com/cloudevents/sdk-go/v2/client"
 	"github.com/cloudevents/sdk-go/v2/event"
 	"github.com/cloudevents/sdk-go/v2/protocol"
 	"github.com/cloudevents/sdk-go/v2/protocol/http"
 	"go.opencensus.io/plugin/ochttp"
-	"knative.dev/eventing/pkg/adapter/v2/util/crstatusevent"
-	"knative.dev/eventing/pkg/metrics/source"
+
 	duckv1 "knative.dev/pkg/apis/duck/v1"
 	"knative.dev/pkg/tracing/propagation/tracecontextb3"
+
+	"knative.dev/eventing/pkg/adapter/v2/util/crstatusevent"
+	"knative.dev/eventing/pkg/metrics/source"
+	obsclient "knative.dev/eventing/pkg/observability/client"
 )
 
-var newClientHTTPObserved = cloudeventsobsclient.NewClientHTTP
+var newClientHTTPObserved = NewClientHTTPObserved
```

Comment:
nit: maybe `newClientHTTPObserved` is no longer needed and can be replaced with the public `MewClientHTTPObserved`